<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Jongwhwa Park</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1224</width>
    <height>809</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GshSAT (suakii@gmail.com) [2024]</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="programControlGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Port Control</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>192</width>
       <height>53</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="comPortLabel">
          <property name="text">
           <string>COM Port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="baudRateLabel">
          <property name="text">
           <string>BAUD RATE:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QComboBox" name="comPortComboBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="baudRateComboBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>186</width>
       <height>28</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="refreshButton">
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QPushButton" name="connectButton">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="GraphicsLayoutWidget" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>891</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="logTextBrowser">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>570</y>
      <width>891</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="programControlGroup_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Bluetooth Connect</string>
    </property>
    <widget class="QSplitter" name="splitter_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>186</width>
       <height>28</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="btSearchButton">
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="programControlGroup_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Bluetooth Test</string>
    </property>
    <widget class="QPushButton" name="ledOnButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>13 LED On</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ledOffButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>13 LED Off</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
